src/software/thunderscope/field/world_layer.py

```diff
@@ -403,9 +403,10 @@ def draw_ball_state(self, painter, ball_state: BallState):
         painter.setPen(pg.mkPen(Colors.BALL_COLOR, width=LINE_WIDTH))
         painter.setBrush(pg.mkBrush(Colors.BALL_COLOR))
 
-        painter.drawEllipse(
-            self.createCircle(ball_state.global_position, BALL_MAX_RADIUS_METERS)
-        )
+        # Ball should get larger as the height of the ball increases
+        ball_radius = BALL_MAX_RADIUS_METERS * (1 + ball_state.distance_from_ground)
```

Comment:
I think to sell the off the ground effect a bit more, we should multiply the distance from ground by some constant (e.g. 3)
```suggestion
        ball_radius = BALL_MAX_RADIUS_METERS * (1 + 3 * ball_state.distance_from_ground)
```